internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -206,12 +207,10 @@ func (handler StorageHandler) processAsyncPostRequest(writer http.ResponseWriter
 			return
 		}
 
-		log.Debug(fmt.Sprintf("Content Type is '%s' & Media Type is '%s' and Type is '%s'",
-			contentType, deviceResource.Properties.Value.MediaType, deviceResource.Properties.Value.Type))
+		log.Debug(fmt.Sprintf("Content Type is '%s' & Media Type is '%s' and Type is '%s'", strings.ReplaceAll(fmt.Sprintf("%#v", contentType), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", deviceResource.Properties.Value.MediaType), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", deviceResource.Properties.Value.Type), "\"", ""))) // lgtm [go/log-injection]
 
 		if contentType != deviceResource.Properties.Value.MediaType {
-			log.Error(fmt.Sprintf("Incoming reading ignored. Content Type '%s' doesn't match %s resource's Media Type '%s'",
-				contentType, resourceName, deviceResource.Properties.Value.MediaType))
+			log.Error(fmt.Sprintf("Incoming reading ignored. Content Type '%s' doesn't match %s resource's Media Type '%s'", strings.ReplaceAll(fmt.Sprintf("%#v", contentType), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", resourceName), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", deviceResource.Properties.Value.MediaType), "\"", ""))) // lgtm [go/log-injection]
```

Comment:
This line is too long. How about using 2 lines?

Reply:
Of course I also don't like this (unreadable code). I tried it like this, but the LGTM exception should be at the end of the line. 

![image](https://user-images.githubusercontent.com/45031429/146747203-74a0d5bf-4d67-49bb-ad48-4b94c6a8b6f0.png)

Reply:
Unable to test HomeEdge while out of office. The code suggested below doesn't work either?

```suggestion
			log.Error(fmt.Sprintf("Incoming reading ignored. Content Type '%s' doesn't match %s resource's Media Type '%s'", // lgtm [go/log-injection]
			     strings.ReplaceAll(fmt.Sprintf("%#v", contentType), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", resourceName), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", deviceResource.Properties.Value.MediaType), "\"", ""))) // lgtm [go/log-injection]
```

Reply:
I think this should work!